worldedit-core/src/main/java/com/sk89q/worldedit/util/schematic/Schematic.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.util.schematic;
+
+import java.nio.file.Path;
+
+/**
+ * Record representing one Schematic file.
+ */
+public record Schematic(Path path) {
+
+    /**
+     * Get this Schematic's path.
+     * @return This Schematic's path.
+     */
+    public Path getPath() {
+        return path;
```

Comment:
Records generally don't have getters, as they generate the internal `path()` getter for the path object in the constructor